### PR TITLE
Update blackbox testing documentation

### PR DESCRIPTION
**Pull request by lleshchi**

### CODE REVIEW

**Comment by jbaublitz** [comment]
@lleshchi Just a heads up that you'll need to rebase.

File: tests/blackbox/README.md [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
 released, packaged, and installed instance of Stratis is functional.
-Thus the test assumes that the `stratisd` daemon and the `stratis` command
-line are installed, although the test will start the daemon if needed.
 
-Run the test as root and supply 3 or more block devices on the
-command line that are blank for test use, eg.
+Run the tests as root and supply 3 or more block devices on the
+command line that are blank for test use.
+
+The `stratisd` test may be run without the installation of the `stratis` command line.
```
[the remark this answers is not in the document]
Comment (lleshchi) [reply tests/blackbox/README.md]: Should the bash examples be wrapped as well?

[the remark this answers is not in the document]
Comment (mulkieran) [reply tests/blackbox/README.md]: > Should the bash examples be wrapped as well?

I would prefer that each example just uses a single line.

[the remark this answers is not in the document]
Comment (bgurney-rh) [reply tests/blackbox/README.md]: OK, I think this looks good.

**Comment by mulkieran** [comment]
REady for final review.

**Pull request closed without merging**